template/Pages/Home/styles.js

```diff
@@ -216,20 +220,27 @@ export const Line = styled.div`
     position: relative;
     border: 1px solid red;
     align-self: center;
+    margin-bottom: 75px !important;
     &::after{
       content: '';
       position: absolute;
-      border: 2px solid red;
+      border: 3px solid red;
       transform: translateY(-50%);
       right: 100%;
       border-radius: 50%
     }
     &::before{
       content: '';
       position: absolute;
-      border: 2px solid red;
+      border: 3px solid red;
       left: 100%;
       transform: translateY(-50%);
       border-radius: 50%;
     }
+    @media (max-width: 790px) {
```

Comment:
Same here, use mobile first

Reply:
I make the modification in the last commit